rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -198,52 +196,52 @@ def __enter__(self):
     def __exit__(self, exception_type, exception_value, traceback):
         pass
 
-    def set_seed(self,seed=0):
+    def setSeed(self, seed=0):
         return b.setSeed(seed)
 
     @classmethod
-    def create_int_param(self,value=1):
-        return b.CreateIntParameter(value)
+    def createIntParam(self, value=1):
+        return b.createIntParameter(value)
 
     @classmethod
-    def create_float_param(self,value=1):
-        return b.CreateFloatParameter(value)
+    def createFloatParam(self, value=1):
```

Comment:
Generally we will add space before and after operator
but most of the code in this file does not follow this convention

@fiona-gladwin @swetha097 Please check on this once and share your thoughts on this

Reply:
Usually we use this convention in python files